test/integration/dom.css.classes.test.ts

```diff
@@ -117,6 +138,20 @@ describe('Bpmn Elements registry - CSS class management', () => {
       bpmnVisualization.bpmnElementsRegistry.toggleCssClasses(['lane_02', 'gateway_01'], ['class1', 'class3', 'class4']);
       htmlElementLookup.expectLane('lane_02', { label: 'Lane 2', additionalClasses: ['class2', 'class4'] });
       htmlElementLookup.expectExclusiveGateway('gateway_01', { label: 'gateway 1', additionalClasses: ['class4'] });
+
+      // toggle several classes by passing classes in a single string separated with spaces
+      // this is not working (remember this is working for 'add css classes')
+      bpmnVisualization.bpmnElementsRegistry.toggleCssClasses(['gateway_01'], ['extra-class1', 'extra-class2']);
+      htmlElementLookup.expectExclusiveGateway('gateway_01', { additionalClasses: ['class4', 'extra-class1', 'extra-class2'] });
+      bpmnVisualization.bpmnElementsRegistry.toggleCssClasses(['gateway_01'], ['extra-class1 extra-class2']);
+      htmlElementLookup.expectExclusiveGateway('gateway_01', { additionalClasses: ['class4', 'extra-class1', 'extra-class2', 'extra-class1', 'extra-class2'] });
```

Comment:
❓ Why there is twice the same classes ?

Reply:
Because the first `toogleCssClasses` call created on entry for 'extra-class1' and another one for 'extra-class2' in the CSS registry.
Then, the second `toogleCssClasses` created an entry for 'extra-class1 extra-class2' in the CSS registry.
The array of classes related to the element is `['extra-class1', 'extra-class2', 'extra-class1 extra-class2']`.
This array is then set in the class attribute of the SVG group as a join by space transformation `class4 extra-class1 extra-class2 extra-class1 extra-class2`